build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -63,10 +63,10 @@ public override string ToString()
         // Full versions and package information.
         public string LatestHostBuildMajor => CommitCountString;
         public string LatestHostBuildMinor => "00";
-        public bool EnsureStableVersion => false;
-        public VerInfo LatestHostVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostFxrVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
-        public VerInfo LatestHostPolicyVersion => new VerInfo(1, 2, 0, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
+        public bool EnsureStableVersion { get; set; }
+        public VerInfo LatestHostVersion => new VerInfo(Major, Minor, Patch, ReleaseSuffix, LatestHostBuildMajor, LatestHostBuildMinor, CommitCountString);
```

Comment:
Let's say we are in a release band: `1.2.3-beta`, now the current host package is `1.2.3-beta-0001`. I make a breaking change which people don't want right away, but will pick up eventually in this release. In this case, I would simply bump up the host version to `1.2.4-beta-0002` within the same release `1.2.3-beta`.